tasmota/xdrv_21_wemo_multi.ino

```diff
@@ -390,7 +394,14 @@ private:
 #else
     String setup_xml = FPSTR(WEMO_SETUP_XML);
 #endif
-    setup_xml.replace("{x1", SettingsText(SET_FRIENDLYNAME1 + (_deviceId - 1)));
+    if (_deviceId == 1)
+    {
+      setup_xml.replace("{x1", SettingsText(SET_FRIENDLYNAME1));
+    }
+    else
+    {
+      setup_xml.replace("{x1", SettingsText(SET_FRIENDLYNAME1 + (_deviceId - 1)));
+    }
```

Comment:
This is not needed as in case of a single device ``_deviceId`` is 1 resulting in exactly the same Friendlyname as in your fix.

This adds to code size.